### PR TITLE
MIG-1449: small corrections following review for another branch - OCP 4.12, 4.13, 4.14

### PR DESCRIPTION
**Pull request by anarnold97**
[MIG-1449](https://issues.redhat.com/browse/MIG-1449)

Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14

Issue:

Link to docs preview:

[MTC release notes](https://65910--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes)

QE review:
- [ X] QE has approved this change. - Reviewed by Rayford Johnson

Additional information:

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.---

### CODE REVIEW

File: modules/migration-mtc-release-notes-1-7-06.adoc [hunk 1]
```diff
@@ -10,12 +10,12 @@
 == New features
 
 .Implement proposed changes for DVM support with PSA in Red Hat OpenShift Container Platform 4.12
-With the incoming enforcement of Pod Security Admission (PSA) in {OCP} 4.12, whereby the default pod would run with a `restricted` profile. This `restricted` profile would mean workloads to migrate would be in violation of this policy and no longer work as of now. The above enhancement outlines the changes that will be required to remain compatible with OCP 4.12. (link:https://issues.redhat.com/browse/MIG-1240[*MIG-1240*])
+With the incoming enforcement of Pod Security Admission (PSA) in {OCP} 4.12 the default pod would run with a `restricted` profile. This `restricted` profile would mean workloads to migrate would be in violation of this policy and no longer work as of now. The following enhancement outlines the changes that will be required to remain compatible with OCP 4.12. (link:https://issues.redhat.com/browse/MIG-1240[*MIG-1240*])
```
Comment (maxwelldb) [inline modules/migration-mtc-release-notes-1-7-06.adoc:13]: "would" or "will"?

**Review by maxwelldb — COMMENTED** [review]
Left a comment. 

For merge review, make sure the preview is updated. Doesn't seem to be the case right now.

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**